subprojects/maven/src/main/java/org/gradle/api/publish/maven/MavenPom.java

```diff
@@ -152,6 +153,22 @@
     @Incubating
     void mailingLists(Action<? super MavenPomMailingListSpec> action);
 
+    /**
+     * Configures the properties for the publication represented by this POM.
+     *
+     * @since 5.3
+     */
+    @Incubating
+    void setProperties(MapProperty<String, String> properties);
```

Comment:
There shouldn't be a setter.  We only need to expose the Property/Provider getter